Playbooks/Enrich-SentinelIncident-MDATPTVM/azuredeploy.json

```diff
@@ -1,43 +1,25 @@
 {
-    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
     "contentVersion": "1.0.0.0",
-    "metadata":{
-        "comments": "This playbook will enrich the Client machine that is part of sentinel incident with thread vulnerabilities data (TVM) with CVE that their score is grater then 7.5. Also it automatically add this information to the incident as comments and change the incident severity to High.",
-        "author": "Yaniv Shasha"
-    },
     "parameters": {
         "PlaybookName": {
             "defaultValue": "Enrich-SentinelIncident-MDATPTVM",
             "type": "String"
         }
     },
-    "variables": {
-         "AzureSentinelConnectionName": "[concat('azuresentinel-', parameters('PlaybookName'))]"
-    },
+    "variables": {},
     "resources": [
-        {
-            "type": "Microsoft.Web/connections",
-            "apiVersion": "2016-06-01",
-            "name": "[variables('AzureSentinelConnectionName')]",
-            "location": "[resourceGroup().location]",
-            "properties": {              
-                "customParameterValues": {},
-                "api": {
-                    "id": "[concat('/subscriptions/', subscription().subscriptionId, '/providers/Microsoft.Web/locations/', resourceGroup().location, '/managedApis/azuresentinel')]"
-                }
-            }
-        },
         {
             "type": "Microsoft.Logic/workflows",
             "apiVersion": "2017-07-01",
             "name": "[parameters('PlaybookName')]",
-            "location": "[resourceGroup().location]",
+            "location": "[resourceGroup().location",
```

Comment:
Hi @noapocalypse Please keep the location for workbook as "[resourceGroup().location]", thanks!

Reply:
done